external-providers/java-external-provider/pkg/java_external_provider/provider.go

```diff
@@ -347,18 +347,49 @@ func (p *javaProvider) Init(ctx context.Context, log logr.Logger, config provide
 		}
 	}
 
-	args := []string{
+	jdtlsBasePath, err := filepath.Abs(filepath.Dir(filepath.Dir(lspServerPath)))
+	if err != nil {
+		return nil, additionalBuiltinConfig, fmt.Errorf("failed finding jdtls base path - %w", err)
+	}
+
+	sharedConfigPath, err := getSharedConfigPath(jdtlsBasePath)
+	if err != nil {
+		return nil, additionalBuiltinConfig, fmt.Errorf("failed to get shared config path - %w", err)
+	}
+
+	jarPath, err := findEquinoxLauncher(jdtlsBasePath)
+	if err != nil {
+		return nil, additionalBuiltinConfig, fmt.Errorf("failed to find equinox launcher - %w", err)
+	}
+
+	javaExec, err := getJavaExecutable(true)
+	if err != nil {
+		return nil, additionalBuiltinConfig, fmt.Errorf("failed getting java executable - %v", err)
+	}
+
+	jdtlsArgs := []string{
+		"-Declipse.application=org.eclipse.jdt.ls.core.id1",
 		"-Djava.net.useSystemProxies=true",
-		"-configuration",
-		"./",
-		//"--jvm-arg=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:1044",
```

Comment:
I'd say lets not remove this, as it is quite useful for debugging the bundle. I use it constantly

Reply:
fixed now